Libraries/Image/ImageBackground.js

```diff
@@ -42,11 +43,24 @@ const View = require('View');
  * ```
  */
 class ImageBackground extends React.Component {
+  setNativeProps(props: Object) {
+    if (this._viewRef) {
+      ensureComponentIsNative(this._viewRef);
+      this._viewRef.setNativeProps(props);
+    }
+  }
+
+  _viewRef: View;
+
+  _captureRef = ref => {
+    this._viewRef = ref;
+  };
+
   render() {
     const {children, style, imageStyle, imageRef, ...props} = this.props;
 
     return (
-      <View style={style}>
+      <View style={style} ref={this._captureRef}>
```

Comment:
N00b question: Will this conflict with ref prop existed in this.props (<ImageBackground ref={...}>...</ImageBackground>)?